bitnami/clickhouse/templates/pdb.yaml

```
@@ -0,0 +1,34 @@
+{{- /*
+Copyright Broadcom, Inc. All Rights Reserved.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if .Values.pdb.create }}
+{{- $shards := .Values.shards | int }}
+{{- range $i, $e := until $shards }}
+apiVersion: {{ include "common.capabilities.policy.apiVersion" $ }}
+kind: PodDisruptionBudget
+metadata:
+  name: {{ printf "%s-shard%d" (include "common.names.fullname" $ ) $i }}
```

What about this change? 

```suggestion
  name: {{ printf "%s-shard-%d" (include "common.names.fullname" $ ) $i }}
```

---

Thanks for the suggestion, I followed same approach as [here](https://github.com/bitnami/charts/blob/0c0e657f1f5b2ecc1d33a60ee09b57e12c0a5dc6/bitnami/clickhouse/templates/statefulset.yaml#L11), that's the reason why there is no `-` there